week-5/C-promises/exercise.js

```diff
@@ -58,9 +81,23 @@ function exercise4() {
   element
 */
 
+
+
 function exercise5() {
-  // Write your implementation here
-}
+  var pEx5 = document.querySelector("#exercise5");
+  var promise = new Promise(function (resolve) {
+    resolve("Hello Promise")
```

Comment:
Nice work!

There's also a shorter way of writing a resolved Promise. Do you know what that is? It is shorter & easier to write, however it is less flexible that the method you used here.